src/vrdev/src/Scatterplot.ts

```diff
@@ -0,0 +1,57 @@
+//run tsc Scatterplot.ts to compile into Scatterplot.js file
+
+// for running on browser
+// import * as _d3 from "d3";
+
+// declare global {
+//   const d3: typeof _d3;
+// }
+
+//for unit testing
+import * as d3 from "d3";
+
+
+export class Scatterplot{
+    data: number[];
+    shape: string;
+
+  constructor(){
+    this.data = [0, 0];
+  }
+  init(data: number[]){
```

Comment:
I think this should take as input a container element:

init(container: HTMLElement, data: number[]);

Then all of the d3 selection functions can be called on that container. Otherwise, d3 will search the entire document each time.